coins/zcl.json

```diff
@@ -5,6 +5,8 @@
     "requireShielding": true,
     "peerMagic": "24e92764",
     "txfee": 0.0004,
+    "sapling": 476969,
+    "overwinter": 476969,
```

Comment:
```suggestion
```
This value isn't actually required. Can you remove it?

Reply:
Actually, you can set `sapling` to just true since that height has passed.

Reply:
Yes right :) At the time it was not